assets/sass/components/global/_googlesitekit-side-sheet.scss

```diff
@@ -0,0 +1,60 @@
+/**
+ * Side Sheet styles.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-side-sheet {
+		background-color: $c-surfaces-surface;
+		bottom: 0;
+		overflow: hidden;
+		position: fixed;
+		right: -100%;
+		top: 0;
+		transition: right 1s;
+		width: 100%;
+		z-index: 10002;
```

Comment:
Please use add a new variable to the list which was just added 👍 Also double check that the value makes sense relative to the others.
https://github.com/google/site-kit-wp/blob/c0c351575827676bb49b2a0b9c26274632f4d183/assets/sass/config/_variables.scss#L315-L329

Reply:
Applied, thank you!